tests/php/Stub/Null/Method.php

```diff
@@ -58,7 +58,7 @@ public function getRegisterHandler(): RegisterHandlerInterface
      */
     public function getThumbnail(): string
     {
-        // TODO: Implement getThumbnail() method.
+        return 'null';
```

Comment:
```suggestion
        return '';
```
Or if the string 'null' _is_ somehow correct, please explain how in a comment in the code.
Same with any other method you've added this as the return value.

Reply:
Updated